### PR TITLE
some improvements

### PR DESCRIPTION
**Pull request by doom-fr**
- simplification of the example (no need to give the path : use glob)
- change example import to work everywhere
- improvement of documentation (change also pip installation to setup.py installation)
- clear of __init__.py

I have made this changes as it does not work as is on my computer.

See, and keep what you want !

### CODE REVIEW

**Comment by doom-fr** [comment]
go back to pip installation !

File: pypowermate/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-from powermate import Powermate, PowermateTimeoutException
```
Comment (blastur) [inline pypowermate/__init__.py:1]: Why?

File: examples/pulse_and_brightness.py [hunk 2]
```diff
@@ -2,39 +2,58 @@
...
 
 if __name__ == '__main__':
-	if len(sys.argv) != 2:
-		sys.stderr.write('usage: %s <input device>\n' % sys.argv[0])
+	if len(sys.argv) > 1 :
+		sys.stderr.write('usage: %s \n  the command has no argument' % sys.argv[0])
+		sys.exit(1)
+	powermates=glob.glob('/dev/input/by-id/*PowerMate*')
+	if len(powermates) == 0 :
+		print('no powermate found !')
```
Comment (blastur) [inline examples/pulse_and_brightness.py]: I'd prefer if you can still pass paths on the cmdline. Perhaps the glob could be used only if no parameters are passed?

File: pypowermate/__init__.py (hunk 1 above)
Comment (doom-fr) [reply pypowermate/__init__.py:1]: Sorry for the delay.

So If there is `from powermate import Powermate, PowermateTimeoutException`, error is :
```Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
>>> import pypowermate
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ratnayake/Documents/projets/audionet/powermate/pypowermate-master/pypowermate/__init__.py", line 1, in <module>
    from powermate import Powermate, PowermateTimeoutException
ImportError: No module named 'powermate'
```


If there is `from pypowermate import Powermate, PowermateTimeoutException`, error is :
```Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
>>> import pypowermate
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/powermate/pypowermate-master/pypowermate/__init__.py", line 1, in <module>
    from pypowermate import Powermate, PowermateTimeoutException
ImportError: cannot import name 'Powermate'
```


If there is `#nothing`, no error :
```Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
>>> import pypowermate
>>>
```
It's is quite not perfect, but it works.


Think the solution is to put `from pypowermate.powermate import Powermate, PowermateTimeoutException` in __init.py :
```Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
>>> import pypowermate 
>>> p=pypowermate.Powermate('/dev/input/by-id/usb-Griffin_Technology__Inc._Griffin_PowerMate-event-if00')
>>>
```

Work fine !

**Comment by doom-fr** [comment]
I have updated this pull request.
Tell me if it is cool or not.

**Comment by doom-fr** [comment]
I let you deal with versioning.

File: pypowermate/__init__.py (hunk 1 above)
Comment (blastur) [reply pypowermate/__init__.py:1]: This was caused by an incompatibility Python 2 vs 3, it's now fixed on the master branch.

File: examples/pulse_and_brightness.py (hunk 2 above)
Comment (blastur) [reply examples/pulse_and_brightness.py]: This was added to the master branch, thanks for the contribution.

**Pull request closed without merging**